cli/tests/command_executor/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#   Copyright 2011-2015 Quickstep Technologies LLC.
+#   Copyright 2015 Pivotal Software, Inc.
```

Comment:
Since this is a new file, please remove unrelated copyrights, and add the university one.